app/src/main/res/layout/dialog_alert_import_summary.xml

```diff
@@ -1,30 +1,46 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:orientation="vertical"
-    android:paddingBottom="10dp">
+    android:layout_height="wrap_content">
 
-    <TableLayout
-        android:id="@+id/summary_table"
-        android:layout_width="fill_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginLeft="8dp"
-        android:layout_marginRight="8dp"
-        android:stretchColumns="2" />
+    <androidx.core.widget.NestedScrollView
+        android:id="@+id/scroll_content_frame"
+        android:layout_width="match_parent"
+        android:layout_height="210dp"
+        app:layout_behavior="@string/appbar_scrolling_view_behavior">
+d
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical"
+            android:paddingBottom="10dp">
 
-    <Button
-        android:id="@+id/summary_details_btn"
-        style="?android:attr/buttonStyle"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center_horizontal"
-        android:layout_marginLeft="10dp"
-        android:layout_marginTop="10dp"
-        android:layout_marginRight="10dp"
-        android:layout_marginBottom="3dp"
-        android:text="@string/check_preferences_details_btn"
-        android:textColor="@color/colorTreatmentButton"
-        android:visibility="gone" />
+            <TableLayout
+                android:id="@+id/summary_table"
+                android:layout_width="fill_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="8dp"
+                android:layout_marginEnd="8dp"
+                android:stretchColumns="2" />
 
-</LinearLayout>
\ No newline at end of file
+            <Button
+                android:id="@+id/summary_details_btn"
+                style="?android:attr/buttonStyle"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_gravity="bottom|center_horizontal"
+                android:gravity="bottom"
```

Comment:
`android:gravity="bottom"` causing v-align of button issue (as seen below), removing that property will fix it

<img src="https://user-images.githubusercontent.com/5094588/94742170-8c031000-0375-11eb-9d4a-7171b0656e26.JPG" height="160">

Reply:
fixed the issues .